tracing-futures/src/lib.rs

```diff
@@ -419,12 +419,13 @@ impl<T: crate::stdlib::future::Future> crate::stdlib::future::Future for WithDis
 
 #[cfg(feature = "std")]
 impl<T> WithDispatch<T> {
+    /// Wrap a future, stream, sink or executor with the same subscriber as this WithDispatch.
     #[cfg(any(feature = "tokio", feature = "tokio-alpha", feature = "futures-03"))]
     #[cfg_attr(
         docsrs,
         doc(cfg(any(feature = "tokio", feature = "tokio-alpha", feature = "futures-03")))
     )]
```

Comment:
If we're making this public, we should probably remove the feature flags — this method was feature-flagged when it was `pub(crate)`, since it was only _used_ when that set of features were enabled, and was dead code otherwise. However, if it's a public API, `futures` 0.1 users should be able to use it as well.

Reply:
ok, I'll remove these.